lib/smart_proxy_dhcp_infoblox/plugin_configuration.rb

```diff
@@ -16,7 +16,7 @@ def load_dependency_injection_wirings(c, settings)
       c.dependency :connection, (lambda do
                                   ::Infoblox.wapi_version = '2.0'
                                   ::Infoblox::Connection.new(:username => settings[:username] ,:password => settings[:password],
-                                                             :host => settings[:server], :ssl_opts => {:verify => false})
+                                                             :host => settings[:server], :ssl_opts => { :verify => true }, :logger => ::Proxy::LogBuffer::Decorator.instance)
```

Comment:
I also set logger which leaves nice INFO and DEBUG messages done by faraday gem. This will be quite useful for troubleshooting later on.